hdlConvertor/toPy.cpp

```diff
@@ -27,7 +27,10 @@ ToPy::ToPy() {
 	}
 	auto import = [this](PyObject *&obj, const std::string &name) {
 		obj = PyObject_GetAttrString(hdlAst_module, name.c_str());
-		assert(obj != NULL &&"Bug in this library hdlConvertor.hdlAst not as expected from C");
+		if (nullptr == obj) {
+			std::cout << "Bug in this library hdlConvertor.hdlAst not as expected from C\n";
```

Comment:
First thing which comes in my mind is:
string err_msg = "<msg>" + name;
PyErr_SetString(PyExc_AssertionError, err_msg.c_str());
https://github.com/Nic30/hdlConvertor/blob/3c1fbae9bb1a4e03ede9e9b2806624d12b1c5b90/hdlConvertor/_hdlConvertor.pyx#L150 but it is required to check object consistency after constructor is used and use "raise" if something went wrong because using c++ exception in constructor is also not a good option.

Reply:
Yeah, that was some debug code that snuck in to my commit.  I'll remove it.  Or I could leave the extra messaging in (nice when debugging!) and keep the assert so it fails.  This really should never happen except during development.